src/Core/Module/ModuleRepository.php

```diff
@@ -261,24 +266,33 @@ private function getModuleDatabaseAttributes(string $moduleName): array
         return $this->installedModules[$moduleName] ?? [];
     }
 
+    /**
+     * @return array
+     */
+    private function getModulesFromHook()
+    {
+        if ($this->modulesFromHook === null) {
+            $modulesFromHook = $this->hookManager->exec('actionListModules', [], null, true);
+            $modulesFromHook = array_values($modulesFromHook ?? []);
+            $this->modulesFromHook = empty($modulesFromHook) ? $modulesFromHook : array_merge(...$modulesFromHook);
```

Comment:
I know it was already handled this way in previous code but I don't understand why we use `array_values` and even less the `array_merge` with the `...` variadic operator 🤔